content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -291,6 +290,12 @@ please see [here](https://kubernetes.io/docs/concepts/cluster-administration/net
 ```shell
 kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/v0.10.0/Documentation/kube-flannel.yml
 ```
+Note that `flannel` works on `amd64`, `arm`, `arm64` and `ppc64le`, but until version `0.11.0` is released
+you need to use the following manifest that supports all the architectures:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/c5d10c8/Documentation/kube-flannel.yml
```

Comment:
Shall we use `https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml` instead of a fixed commit?

Users may get confused why using this commit, does it have any special meanings and etc.

Reply:
i did this intentionally. the thing about using HEAD here is that they might decide to break the manifest somewhere before 0.11.0. on the other hand we have a commit that introduces the fat manifest and we had someone verify that it even works for multi-arch node clusters.

so i'd prefer we keep this specific commit and hope that 0.11.0 is released so that we can use a tag instead.

Reply:
Makes sense :+1: